modules/configuring-default-seccomp-profile.adoc

```diff
@@ -1,15 +1,75 @@
 [id="configuring-default-seccomp-profile_{context}"]
-= Configuring the default seccomp profile
-OpenShift ships with a default seccomp profile that is referenced as `runtime/default`. You can enable the default seccomp profile for a pod or container workload by setting `RuntimeDefault` as following:
+= Enabling the default seccomp profile for all pods
 
-.Example
+{product-title} ships with a default seccomp profile that is referenced as `runtime/default`. You can enable the default seccomp profile for a pod or container workload by creating a custom Security Context Constraint (SCC), editing this and adding the default seccomp profile to this custom SCC. Finally add the custom SCC to the default service account. 
 
+[NOTE]
+====
+There is a requirement to create a custom SCC because editing the default SCCs can lead to issues when some of the platform pods deploy or {product-title} is upgraded. For more information, see the section entitled "Default security context constraints". 
+====
+
+Follow these steps to enable the default seccomp profile for all pods:
+
+. Export the available `restricted` Security Context Constraints (SCC) to a yaml file: 
++
+[source, terminal]
+----
+$ oc get scc restricted -o yaml > restricted-seccomp.yaml
+----
+
+. Edit the created `restricted` SCC yaml file: 
++
+[source, terminal]
+----
+$ vi restricted-seccomp.yaml
+----
+
+. Update as shown in this example: 
++
 [source, yaml]
 ----
-spec:
-  securityContext:
-    seccompProfile:
-      type: RuntimeDefault
+kind: SecurityContextConstraints
+metadata:
+  name: restricted  <1> 
+<..snip..>
+seccompProfiles:    <2> 
+- runtime/default   <3> 
+----
+<1> Change to `restricted-seccomp`
+<2> Add `seccompProfiles:`
+<3> Add `-` `runtime/default`
+
+. Create the custom SCC: 
++
+[source, terminal]
+----
+$ oc create -f restricted-seccomp.yaml
+----
++
+.Expected output
++
+[source, terminal]
+----
+securitycontextconstraints.security.openshift.io/restricted-seccomp created
+----
+
+. Add the custom SCC to the ServiceAccount: 
++
+[source, terminal]
+----
+$ oc adm policy add-scc-to-user restricted-seccomp -z default
+----
++
+[NOTE]
+====
+The `default` service account is the one taking the responsibility of running the pods, and OpenShift uses and injects this service account into every pod that is launched. 
```

Comment:
This is incorrect. That ServiceAccount will be used unless the user configures a different one. Also, it's not OpenShift injecting it. This is default K8s behavior, nothing specific for OCP.